### ISSUE TITLE
Modifiers are a bit inconsistent

### ISSUE THREAD

**Issue by RCrockford** [RealityMachina/Basic-Panic-System#8]
I'm not sure if this is intentional, but the modifiers are used differently in panic rolls and eject rolls.

Panic rolls use them as percentages scaled down to D20 roll.

Eject rolls use them scaled up by 5, meaning that you can end up with a modifier of over 500 on a 100 roll. Seems to mean that once a mech is rolling for panic they are just doing a 50/50 almost all the time. I've noticed enemy mechs seem a bit trigger happy on the eject once their status goes low.

**Comment by RealityMachina** [comment]
Yeah, intentional balance decision with the default json values: the design intent is that generally once they're panicking, they're in a really bad state and should be likely to consider pulling out.

RogueTech has an example of less extreme values if you want a more toned down version.
